UnitTests/Catalog/Services/CommentsRepository/AddCommentTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Target = Catalog.Services.Comments.CommentsRepository;
+using Common.Interfaces;
+using Common.Models.Comments;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Threading.Tasks;
+
+namespace UnitTests.Catalog.Services.CommentsRepository
+{
+    [TestClass]
+    public class AddCommentTests
```

Comment:
Not all the test cases are covered for this method:
- Comment with invalid Product ID
- Comment with an empty message
Probably, I've missed a couple of them as well.
I understand that we will work with unit tests extensively later, covering our logic more thoroughly. But this is a bad practice to leave unit tests for the uncertain future.